client/src/containers/App/App.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { provideInjector } from 'lib/Injector';
-import { renderRoutes } from 'react-router-config';
+import { Outlet } from 'react-router';
 
 /**
  * Empty container for the moment, will eventually contain the CMS menu`
  * and apply to document.body, rather than just one specific DOM element.
  */
 const App = ({ route }) => (
-  <div className="app">{renderRoutes(route.routes())}</div>
+  <div className="app"><Outlet /></div>
```

Comment:
Rendering the routes is no longer the responsibility of this component. It's handled by `BootRoutes.js` explicitly.